airflow/www/fab_security/sqla/manager.py

```diff
@@ -339,7 +333,7 @@ def filter_roles_by_perm_with_action(self, action_name: str, role_ids: List[int]
             )
         ).all()
 
-    def get_role_permissions_from_db(self, role_id: int) -> List[Permission]:
+    def get_permissions_from_roles(self, role_id: int) -> List[Permission]:
```

Comment:
```suggestion
    def get_permissions_from_role(self, role_id: int) -> List[Permission]:
```
Should be singular given it takes a single role_id. (Possibly also called `_from_role_id`, but don't mind on that one)

Reply:
Its actually not used anymore so should be removed entirely